test/python/pulse/test_pulse_lib.py

```diff
@@ -314,6 +325,52 @@ def check_drag(duration, sigma, amp, beta):
         with self.assertRaises(PulseError):
             check_drag(duration=50, sigma=4, amp=0.8, beta=-20)
 
+    def test_trig_pulses(self):
```

Comment:
Please write one dedicated function for one symbolic pulse class (for ease of debugging)

Reply:
Thank you.